docs/design/proposals/cephfs-fscrypt.md

```diff
@@ -167,59 +168,63 @@ graph LR
 
 ```
 
-The diagram shows the keys flowing from Ceph CSI to the
-Kernel API unlocking a directory. On the way key material from
-Ceph-CSI passes two key derivation steps in fscrypt: *protectors* and
-*policies*.
+The diagram shows the keys flowing from ceph-csi to the Kernel API unlocking
+a directory. On the way, key material from ceph-csi passes two key derivation
+steps in fscrypt:
+
+- *protectors* and
+- *policies*
 
 
-`fscrypt` supports multiple *protectors*, which may source secrets from login
-passwords, custom passwords or soon Ceph-CSI. Unlocking a protector
+`fscrypt` supports multiple *protectors*. These may source secrets from login
+passwords, custom passwords or soon ceph-csi. Unlocking a protector
 yields a *protector key* that is then used to unlock a *policy*.
 
 A *policy* may unlock multiple directories. In our case there will be
-only a single policy for a single well know directory on the
-subvolume root. A policy is used to derive a *policy key*, which is
-finally passed to the kernel API along settings like the
-desired encryption algorithm.
-
-Going back to the beginning of the diagram and the interface between
-`fscrypt` and Ceph-CSI. Data encryption key (DEK) styles metadata
-and integrated require different protectors. The `fscrypt` protector
-key sources `CustomPasswordSource` and `RawKeySource` in the following
-differ in how they derive a key from a source. Please refer to the
-`fscrypt` design doc for details.
-
-Metadata DEKs: In the RBD case, Ceph-CSI stores a wrapped key in the
-RBD volume metadata. A user configured secret (e.g a Kubernetes
+only a single policy for a single well-known directory on the sub-volume root.
+A policy is used to derive a *policy key*, which is passed to the Kernel API
+along with other settings, such as the desired encryption algorithm.
+
+_Going back to the beginning of the diagram, looking at the interface between
```

Comment:
Marcel - this is the one sentence I didn't parse properly. You can see how I tried to rephrase it, but it didn't feel right.